go.mod

```diff
@@ -5,16 +5,15 @@ go 1.13
 require (
 	github.com/ONSdigital/dp-api-clients-go v1.10.1
 	github.com/ONSdigital/dp-authorisation v0.1.0
-	github.com/ONSdigital/dp-dataset-api v1.20.0 // indirect
 	github.com/ONSdigital/dp-healthcheck v1.0.4
 	github.com/ONSdigital/dp-net v1.0.3
 	github.com/ONSdigital/dp-rchttp v1.0.0
```

Comment:
can we remove this dependency, please? dp-rchttp was migrated to dp-net, and running a `go mod tidy`, I see that this dependency disappears. Maybe you can just run `go mod tidy` and commit go.mod and go.sum